spec/models/rules_of_origin/scheme_spec.rb

```diff
@@ -39,6 +40,22 @@
     it { expect(links.first).to have_attributes url: 'https://www.hmrc.gov.uk' }
   end
 
+  describe '#cumulation_methods=' do
+    subject(:cumulation_methods) { instance.cumulation_methods }
+
+    before { instance.cumulation_methods = data }
+
+    let(:instance) { described_class.new }
+
+    let(:expected_response) { { 'bilateral' => %w[GB CA], 'extended' => %w[EU AD], 'diagonal' => %w[EU AD] } }
+
+    let(:data) do
+      { 'bilateral' => { 'countries' => %w[GB CA] }, 'extended' => { 'countries' => %w[EU AD] }, 'diagonal' => { 'countries' => %w[EU AD] } }
+    end
+
+    it { is_expected.to eq expected_response }
```

Comment:
testing Hash's with `eq` can be error prone, since Ruby 1.9 Hashes have maintained order, hence I think

```ruby
expect({ 'one' => 1, 'two' => 2 }).to eq({'two' => 2, 'one' => 1})
```

will actually fail even though you expect it to pass

I know in general RSpec recommends use of the `includes` matches